src/main/io/beeper.c

```diff
@@ -178,7 +178,7 @@ static uint8_t beep_multiBeeps[MAX_MULTI_BEEPS + 1];
 #define BEEPER_WARNING_BEEP_GAP_DURATION 10
 
 // Beeper off = 0 Beeper on = 1
```

Comment:
If we change this to a boolean, maybe the comment has no sense

Reply:
Good point!